src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -936,7 +938,56 @@ private protected override IRowMapper MakeRowMapper(DataViewSchema schema)
             => new Mapper(this, schema);
     }
 
-    /// <include file='doc.xml' path='doc/members/member[@name="LightLDA"]/*' />
+    /// <summary>
+    /// The LDA transform implements <a href="https://arxiv.org/abs/1412.1576">LightLDA</a>, a state-of-the-art implementation of Latent Dirichlet Allocation.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes |
+    /// | Input column data type | [key](xref:Microsoft.ML.Data.KeyDataViewType) data types|
+    /// | Output column data type | Vector or <xref:System.Single>|
+    ///
+    ///  Latent Dirichlet Allocation is a well-known [topic modeling](https://en.wikipedia.org/wiki/Topic_model) algorithm that infers semantic structure from text data,
+    ///  and ultimately helps answer the question on "what is this document about?".
+    ///  It can be used to featurize any text fields as low-dimensional topical vectors.
+    ///  LightLDA is an extremely efficient implementation of LDA developed in MSR-Asia that incorporates a number of
+    ///  optimization techniques.
+    ///  With the LDA transform, ML.NET users can train a topic model to produce 1 million topics with 1 million vocabulary
+    ///  on a 1-billion-token document set one a single machine in a few hours(typically, LDA at this scale takes days and requires large clusters).
+    ///  The most significant innovation is a super-efficient O(1) [Metropolis-Hastings sampling algorithm](https://en.wikipedia.org/wiki/Metropolis–Hastings_algorithm),
+    ///  whose running cost is agnostic of model size, allowing it to converges nearly an order of magnitude faster than other [Gibbs samplers](https://en.wikipedia.org/wiki/Gibbs_sampling).
+    ///
+    ///  In an Ml.Net pipeline, this estimator requires the output of some preprocessing, as its input.
+    ///  A typical pipeline operating on text would require performing text normalization, tokenization and producing n-grams to than supply to LDA.
+    ///  See the example usage in the SeeAlso section for usage suggestions.
+    ///
+    ///  If we have the following three lines of text, as data points:
+    ///  * I like to eat bananas.
+    ///  * I eat bananas everyday.
+    ///  * LightLDA improves the sampling throughput and convergence speed via a novel O(1) metropolis-Hastings sampler,
+    ///  and allows a small cluster of machines to tackle very large data and model sizes based on the model scheduling
+    ///  and data parallelism capabilities of the DMTK parameter server.(quoted from [LightLDA](http://www.dmtk.io/lightlda.html))
```

Comment:
run-on sentence, can this be reworked?

Reply:
it is just an example sentence.. but i see how it can be confusing. Let me pick something else. 

---
In reply to: [277109187](https://github.com/dotnet/machinelearning/pull/3442#discussion_r277109187) [](ancestors = 277109187)